docs/Getting-Started.md

```diff
@@ -67,6 +67,8 @@ Fastify offers an easy platform that helps solve all of problems, and more.
 > })
 > ```
 >
+> Similarly, specify `::1` to accept only local connections via IPv6, or `::` to accept any connection on all IPv4 and IPv6 addresses simultaneously.
```

Comment:
The `::` may or _may not_ open the socket on all IPv4 addresses in addition to the IPv6 addresses. It is OS dependent according to the core Node documentation. Therefore, this statement is not complete.

Reply:
TIL, thanks for catching that.